primitives/Cargo.toml

```diff
@@ -13,13 +13,13 @@ targets = ["x86_64-unknown-linux-gnu"]
 [dependencies]
 codec = { package = "parity-scale-codec", version = "2.0.0", default-features = false, features = ["derive"] }
```

Comment:
other Cargo.toml have upgraded to 2.3.1, would better also change here.

Reply:
upgrade to 2.3.1